ding/policy/dqn.py

```diff
@@ -84,6 +95,12 @@ class DQNPolicy(Policy):
         discount_factor=0.97,
         # (int) The number of step for calculating target q_value
         nstep=1,
+        model=dict(
+            #(list(int)) Sequence of ``hidden_size`` of subsequent conv layers and the final dense layer.
+            encoder_hidden_size_list=[128, 128, 64],
+            # (bool) whether enable dueling head
```

Comment:
Each comment line should start from upper case and end with a period

Reply:
fixed all